Assets/Scripts/Trigger/buttonTrigger.cs

```diff
@@ -42,5 +42,20 @@ public class buttonTrigger : Trigger {
 		}
 	}
 
+	public override void interact()
+	{
+		UITrigger secretItem = GetComponent<UITrigger> ();
+		if (secretItem != null && !secretItem.found) {
+			secretItem.interact ();
+			buttonPressed = secretItem.replacementSprites [0];
```

Comment:
If you believe it's better to keep it here then sure. If you fix the input manager thing then you can merge.